include/UUID.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+** EPITECH PROJECT, 2024
+** Plazza
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#pragma once
+#include <uuid/uuid.h>
+#include <string>
+
+#ifndef _UUID_STRING_T
+    typedef char uuid_string_t[37];
```

Comment:
you forgot the `#define _UUID_STRING_T`
```suggestion
    #define _UUID_STRING_T
    typedef char uuid_string_t[37];
```